### ISSUE TITLE
Incoming WA messages are repeated.

### ISSUE THREAD

**Issue by ajiragroup** [frappe/crm#360]
I hope this speaks itself.
![image](https://github.com/user-attachments/assets/e61875da-4dcd-47a5-88b4-2f3c0a25cc61)

I assure you, I had sent only 1 message of each text.

![image](https://github.com/user-attachments/assets/12bdc6ba-a638-4168-a4ca-f4695993d581)

**Comment by shariquerik** [comment]
@ajiragroup I tested I was not able to replicate the issue. Check the video

https://github.com/user-attachments/assets/b904e897-75fc-4b21-ade2-333f72139a2c

**Comment by ajiragroup** [comment]
hmm. we had tested in a fresh installation. so there could be no other issues. Let me try again in the latest version.

**Comment by shariquerik** [comment]
Closing since it is not reproducible
